test/lint/lint-files.py

```diff
@@ -19,7 +19,7 @@
 ALLOWED_FILENAME_REGEXP = "^[a-zA-Z0-9/_.@][a-zA-Z0-9/_.@-]*$"
 ALLOWED_SOURCE_FILENAME_REGEXP = "^[a-z0-9_./-]+$"
 ALLOWED_SOURCE_FILENAME_EXCEPTION_REGEXP = (
-    "^src/(secp256k1/|univalue/|test/fuzz/FuzzedDataProvider.h)"
+    "^src/(secp256k1/|univalue/|test/fuzz/FuzzedDataProvider.h|simplicity)"
```

Comment:
Should have a slash after `simplicity` like it does for `secp256k1`?

Reply:
I read `/|` as an escape sequence, like `\(`. Not an expert here.